src/Perpetuum/Units/Unit.cs

```diff
@@ -482,9 +482,21 @@ protected virtual void DoExplosion()
             Task.Delay(FastRandom.NextInt(0, 3000)).ContinueWith(t => zone.DoAoeDamage(damageBuilder));
         }
 
+        /// <summary>
+        /// Fitted curve to achieve desired explosion distances for bots of certain sizes by class
+        /// Ensures output will be bounded and that arbitrary input will not result in error or undesired output
+        /// </summary>
+        /// <param name="x">SignativeRadius of bot</param>
+        /// <returns>Radius in number of tiles (10m)</returns>
+        private double SigRadiusToExplosionRadius(double x)
+        {
+            x = x.Clamp(3.0, 40.0);
+            return (-3.336453 + 1.617863 * x - 0.09721877 * (x * x) + 0.002064723 * (x * x * x)).Clamp(1.0, 30.0);
```

Comment:
Can we put some meaningful names to these numbers please? :)

i.e. What does -3.336453 represent?

Reply:
This unfortunately is a fitted curve to some key explosion radius outcomes we wanted across different surface sizes.  The coefficients and constants are not going to have much meaning on their own... (at least not that I can articulate)
Which makes this a non-ideal solution to a very undesirable problem as the result of some key balance changes.